### ISSUE TITLE
firebase_messaging - Flutter Android Notification Count is Always null when i send notification from flutter app using http package

### ISSUE THREAD

**Issue by axiftaj** [firebase/flutterfire#10800]
I am sending notifications using dart code with firebase api in my Flutter App and i 'm receiving notification count null. Title and body is receiving. 

I have attached the source code to help with more investigation.
Source code: https://github.com/axiftaj/Flutter-Firebase-Notifications

Flutter Configurations are setup correctly
<img width="728" alt="227712617-075db140-bac8-416e-b1da-542139a24b3c" src="https://user-images.githubusercontent.com/47206155/232439612-6591ee47-aea5-43a9-ad9d-8a29bb548627.png">

![227712573-719c76a5-b1d6-4f4d-a37a-8cadf1620636](https://user-images.githubusercontent.com/47206155/232439724-01392bc5-62be-4544-8180-da84ae38ee51.png)

Dart code for sending notification but you can also check the repository as well.

```
var data = {
              'to' : 'your token',
              'notification' : {
                'title' : 'Asif' ,
                'body' : 'Subscribe to my channel' ,
            },
              'android': {
                'notification': {
                  'notification_count': 23,
                },
              },
              'data' : {
                'foo' : 'bar' ,
              }
            };
```

  ```
  await http.post(Uri.parse('https://fcm.googleapis.com/fcm/send'),
        body: jsonEncode(data) ,
          headers: {
            'Content-Type': 'application/json; charset=UTF-8',
            'Authorization' : 'key=serverkey'
          }
        ).then((value){
          if (kDebugMode) {
            print(value);
          }
        }).onError((error, stackTrace){
          if (kDebugMode) {
            print(error);
          }
        });
```
![231717479-c85adf91-871d-4d8c-9234-1b67bd0fc3fc](https://user-images.githubusercontent.com/47206155/232440310-b5413c34-5ba6-4141-9895-265d71926ac4.png)

PS: This ticket is a dupe of issues 10659, https://github.com/firebase/flutterfire/issues/10659, which you guys closed it and are not responding to my queries. 

It may be because of the label you assign to the issue 10659 and now it's not no longer in your priorities list, that's why i didn't hear back about my latest queries.

**Comment by darshankawar** [comment]
@axiftaj 
Let's follow-up in https://github.com/firebase/flutterfire/issues/10659. I'll reopen it for better tracking.

**Comment by axiftaj** [comment]
> @axiftaj
> Let's follow-up in #10659. I'll reopen it for better tracking.

You guys closed the ticket #10659 without hearing back from me. Do i need to open new issue again.

**Comment by darshankawar** [comment]
@axiftaj 
Per FlutterFire team member's response here: https://github.com/firebase/flutterfire/issues/10659#issuecomment-1512984550, it doesn't seem to be an issue with the plugin, and you may need to reach out as suggested.

Regarding your comment https://github.com/firebase/flutterfire/issues/10659#issuecomment-1513039546 , please understand that there are various factors involved in the plugin and flutterfire plugins are thin wrappers around native SDKs so the interaction happens using Dart. 

> > @axiftaj
> > Let's follow-up in #10659. I'll reopen it for better tracking.
> 
> You guys closed the ticket #10659 without hearing back from me. Do i need to open new issue again.

We listened and looked at your issue and responded as required and suggested what you need to do.
